### ISSUE TITLE
Update Project Profile: Engage

### ISSUE THREAD

**Issue by ashlingam** [hackforla/website#251]
### Overview
Update the following fields for your project. These updated fields will then be updated to be shown on the website.

### Action Items
Provide in the comments below:
- [x] Updated wording, if necessary, on the project card (see [hackforla.org](https://www.hackforla.org/)).
- [x] Information on which tools your team is using for the project (Ex: Figma, Balsamiq, Photoshop, etc.). This is different than programming languages.
- [x] Updates for "Looking For" section.
- [x] Any updated links that are not currently displayed on the site (do you have a demo site?).
- [x] Hero image to be displayed on Project Home Page at 1500 x 700 px.
- [x] Any resources for a Getting Started link (either a link to a wiki or readme).

### Resources/Instructions
See the hero image on the [hackforla.org](https://www.hackforla.org/) homepage.

**Comment by ExperimentsInHonesty** [comment]
- [ ] Added @eselkin to hfla github org. 
- [ ] Sent a slack message, asking him to accept invitation and to let me know, so I can assign him to this issue.

**Comment by eselkin** [comment]
@teddycr, check this
- [x] wording seems fine
- [x] tools: None (we don't have a clickable prototype or anything)
- [x] Looking for: Product Manager, Project Manager (with the focus on growing the project), UX designers, React developers, ML engineer (NLP)
- [x] links look ok
- [x] Hero image is ok
- [x] Wiki hasn't been updated in a minute: https://github.com/hackla-engage/Engage-Product-Documentation/wiki

**Comment by eselkin** [comment]
Actually, our image is small, but I'll look for the original or an alternate.

**Comment by ashlingam** [comment]
@eselkin Could you please provide an ETA on the updated hero image for your project? Thanks!

**Comment by eselkin** [comment]
I couldn't find the original. I'm thinking I'll try to come early tomorrow and take a photo of the city hall or something to replace the one that we have now in a format that fits with your requirements.

**Comment by ashlingam** [comment]
@eselkin Hey Eli, were you able to obtain a hero image for use on your Project Home Page? Thanks

**Comment by eselkin** [comment]
No, I wasn't able to make it last week. Hopefully tomorrow.

**Comment by ExperimentsInHonesty** [comment]
@eselkin At least temporarily we are going to use this image as the hero
![engage-hero](https://user-images.githubusercontent.com/37763229/75752236-29616d00-5cdd-11ea-92bc-7244f304e91f.jpg)

**Comment by eselkin** [comment]
sounds good

**Comment by ExperimentsInHonesty** [comment]
@eselkin can you get Teddy's title for me?  We are going to push this page live tomorrow.

**Comment by eselkin** [comment]
I think we'll have to keep the one that you found, because I can't find our
original... because of a lot of other circumstances I haven't been able to
make it out to Santa Monica to get a new photo.

**Comment by ExperimentsInHonesty** [comment]
This issue is ready to be worked .  only thing remaining is adding the tools to their project card.  Project is not going to provide title for Teddy.

**Comment by KianBadie** [comment]
@ExperimentsInHonesty just want to make sure, the only update right now is to display "Tools: None" on their project home page (I'm assuming the message in parenthesis after "tools: None" in this issue was only to be read in this issue)? I also see that the home page says looking for "NLP Engineers", while this issue says "ML engineer (NLP)." Is that intended?

**Comment by ExperimentsInHonesty** [comment]
@eselkin it says no tools in your file.  Don't you guys use something, like Google Analytics, or Figma, or something else?

**Comment by eselkin** [comment]
I'm not aware of using Figma, I don't think anyone did clickable mocks in anything. Maybe sketch at one point, but I don't have those files. We use hotjar and Google Analytics. These are "tools" not languages etc., right?

**Comment by KianBadie** [comment]
@eselkin sorry for replying to your message late. To answer your question, yes these are tools not languages. The page is checking github repositories for languages. I think Google Analytics would at least qualify to be a tool. I'm not too familiar with hotjar. @ExperimentsInHonesty thoughts?

**Comment by ExperimentsInHonesty** [comment]
@eselkin @KianBadie Yes, hotjar and google analytics would be a tool I would put in the tool list.  We will add them.

**Comment by ExperimentsInHonesty** [comment]
Not sure why but on the [engage project page](https://www.hackforla.org/projects/engage), languages is not showing up.  Who wants to take this issue? Please self assign and post in the slack.  There was recently a pull request on this project, but I think this was broken before that happened.

Please note in your investigation that this repo is not on the hackforla repo.  Public Tree Map is also on its own repo and its working fine.

**Comment by ExperimentsInHonesty** [comment]
Hi @eselkin we updated the project card for engage, but languages didn't show up.  Upon reflection its because we built our website logic to pull from one repo on GitHub.  Since you have many repos, we are going to have to rewrite the logic to accommodate that.  It also affects the contributors that show up.  Engage is not the only project with multiple repos (HelloGov, PublicTreeMap, so this is something we have to do to make all these projects display properly.  Here is the new issue we have opened for this: #458
